src/network.cpp

```diff
@@ -275,6 +275,12 @@ void setupWebServer(BmsRelay *bmsRelay) {
         }
 
         // Set wifi power
+        // add aditional sanity checks, so that the power range is between 9 and
+        // 12 only!
+        if (wifiPower == nullptr || wifiPower->value().toInt() < 9 || wifiPower->value().toInt() > 12) {
+          request->send(400, "text/html", "Wifi Power range MUST be between 9 and 12.");
```

Comment:
According to https://arduino-esp8266.readthedocs.io/en/latest/esp8266wifi/generic-class.html#setoutputpower this can be anywhere from 0 and 20.5 with the docs suggesting that 17.5 is a good sweet spot. Maybe limit this from say 4 to 17?

Reply:
My first intention on this was only to raise the value only little bit, and to not allow a value lower as your hard coded "default". But I would not go as low as 4, since the top shilding of the Batterybox may prevent the Signal from spreading on a value of 4. On a XR the "default" 9 from you was just a little to less...
Therefore I suggest to limit this from say min. 8 to 17 with a default of 9?

Reply:
Sure why not, sounds good.

Reply:
Thanks again for great work!